### PR TITLE
Updates Lanyon for Jekyll 3.0

### PR DESCRIPTION
**Pull request by her**
Fixes a few concatenation bugs, repairs paginate dependencies, and complies with Jekyll's relative permanent link changes in 3.0

### CODE REVIEW

File: _layouts/page.html [hunk 1]
```diff
@@ -4,5 +4,6 @@
 
 <div class="page">
   <h1 class="page-title">{{ page.title }}</h1>
-  {{ content }}
+  <a href="{{ site.baseurl }}{{ page.url }}"></a>
```
Comment (mafagafogigante) [inline _layouts/page.html]: I think this hyperlink is just wrong.

File: _layouts/page.html [hunk 2]
```diff
@@ -4,5 +4,6 @@
...
+    {{ content }}
```
Comment (mafagafogigante) [inline _layouts/page.html]: Looks like you have inconsistent indentation.

File: _layouts/page.html (hunk 1 above)
Comment (wireddown) [inline _layouts/page.html]: @mafagafogigante -- how would you format the link?

I think it's correct :-) See *Rule 1* here: [Configuring Jekyll for User and Project GitHub Pages](http://downtothewire.io/2015/08/15/configuring-jekyll-for-user-and-project-github-pages/)

Comment (mafagafogigante) [inline _layouts/page.html]: I am not questioning the href, but the HTML element. A hyperlink. It is empty. Doesn't serve any purpose on the browser, unless I am missing something.

File: _config.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@
 #
 # Use of `relative_permalinks` ensures post links from the index work properly.
 permalink:           pretty
-relative_permalinks: true
+#relative_permalinks: true
```
Comment (jhabdas) [inline _config.yml]: If this is no longer needed, please consider removing it entirely. It's not a good practice to leave commented out code in source.

**Comment by jhabdas** [comment]
Resolves https://github.com/poole/lanyon/issues/124. Is anyone maintaining this repo?

**Comment by mdo** [comment]
Coming in #238.

**Pull request closed without merging**